traffic_ops/traffic_ops_golang/cdn/cdns.go

```diff
@@ -136,6 +137,8 @@ func (cdn *TOCDN) Read(h http.Header, useIMS bool) ([]interface{}, error, error,
 	// sorting alphabetically by cdn name.
 	if _, ok := cdn.APIInfo().Params["orderby"]; !ok {
 		cdn.APIInfo().Params["orderby"] = "name"
+	} else {
+		return nil, fmt.Errorf("unable to sort the list"), nil, http.StatusInternalServerError, nil
```

Comment:
Doesn't this mean that if I try to set `orderby` myself in the URL it'll give me back an internal server error?

Reply:
Resolved conversation based on offline chat.